test/ContentValidationListenerTest.php

```diff
@@ -1992,14 +1992,6 @@ public function testSaveUnknownDataWhenEmptyInputFilter()
     }
 
     /**
-     * @depends testFailsValidationOfPartialSetsForPatchRequestsForCollectionThatIncludeUnknownInputs
```

Comment:
argh! there's two similarly named test cases, and I evidently landed on the previous, which does not contain the "ForCollection" verbiage. So, yes, probably correct to remove it, though I'm curious if it should be present to test an edge case.